elastic/kibana
# [Console][Legacy Editor] Ace Range implementation not being used

## Summary

After merging this PR: https://github.com/elastic/kibana/pull/46793 a regression was introduced that affects the autocomplete system. This PR fixes it by using the `acequire` to load in the actual `ace` range implementation.

[Update]
## How to review

1. Open Console
2. type in: 
```
POST _search
{
   q[ctrl+space]
}
```
3. Ensure that the body of the request has correctly autocompleted.
